### DISCUSSION TITLE
Documents and MyPC in the Drive List

### DISCUSSION THREAD

**Discussion by lars18th** [doublecmd/doublecmd#1803] [Suggestions]
Hi,

After years using TC, my migration to DC is completing. However, some divergencies in the UI are quite problematic. For sure, I'm not considering to mimmic the entire interface, because this doesn't have sense (DC is a different program). However, some "options" could make more easy the transition, or it could improve the user experience.

Having said that, the suggestion is:

- Add the directories _Documents_ (`home` in Linux) and _MyPC_ (`root` or similar in Linux) in the Drive List... only if the user enables one specific option for it.

I know that we can add these directories to the Directories HotList (CTRL+D), but the idea of adding these to the Drive List could be useful. Or if you prefer, add the option to add up to 3-4 custom directories to this list (and then the user could add manually these directories).

Please, consider it.
Thank you.

**Comment by lars18th** [comment]
Hi,

What's you point of view about this idea: Add in the **Directory Hotlist** the option to mark some entries as **"Drive List"**. This will add these entries at the end of the Drive List.

My objective is to put some specific folders/drives ("My documents" for example) directly in the Drive List. The \<CTRL\>+\<D\> shortcut is useful for a _large_ list of directories. However, I prefer to use the _short_ Drive List for some priority dirs.

Note: The file `src/platform/udrivewatcher.pas` implements the funcionality to create the Drive List. Perhaps some method could be added to include a _small_ (around 3-4) list of user configured entries.

**Comment by antodc** [comment]
What about just putting your favorite folders at the top of the hotlist?

**Reply by lars18th** [reply]
Yes, I now this.

However, I use a lot the keyboard with \<ALT\>+\<F1\> and \<ALT\>+\<F2\>. And this is by far more fast than changing the panel and use the \<CTRL\>+\<D\> to open the Directory Hotlist.

Please, don't confuse. The Directory Hotlist has a lot of sense. However, I has sense as well to add some custom entries to the Drive List.

**Reply by antodc** [reply]
I see only one more tab-click (to switch panel) between the two options, but I understand you are a strong time optimizer. 😄

**Reply by lars18th** [reply]
The difference is that with \<ALT\>+\<F1\> and \<ALT\>+\<F2\> you are going **_directly_** to the Left or Right panel, without the need to check in which panel you are.

For users using the Drive List the Directory Hoslits is secondary. Check for example that in TC are by default "My Documents" and "My PC". And in DC we have the "VFS List". The idea is then to support _custom drives_.
